src/Compilers/CSharp/Test/Syntax/Syntax/SyntaxFactoryTests.cs

```diff
@@ -3,6 +3,7 @@
 // See the LICENSE file in the project root for more information.
 
 #nullable disable
+#pragma warning disable RS0030 // Do not used banned APIs
```

Comment:
If this is needed, then what is the targets file change above doing?

Reply:
Will clean up. I started suppressing on file level but then at some point realized it's too many suppressions would be needed and it's better to avoid adding BannedSymbols.txt in the first place.